index.html

```diff
@@ -59,6 +60,214 @@ <h3 class="connect">LET'S CONNECT</h3>
             </li>
             
           </ul>
+        </section>
+
+            <!--Card One-->
+        <section class="section-1">
+
+          <div class="first-card " >
+            <div class="items">
+              <div>
+                <img src="./images/Snapshoot Portfolio.png" alt="">
+              </div>
+
+              <div>
+                <h1>Tonic</h1>
+              </div>
+
+              <div class="details">
+                <h3>CANOPY</h3>
+                <img src="./images/Counter.png" alt="">
+                <h4>Back End Dev</h4>
+                <img src="./images/Counter.png" alt="">
+                <h4>2015</h4>
+              </div>
+
+              <p>
+                A daily selection of privately<br>personalised reads; no accounts or<br>sign-ups required.
+              </p>
```

Comment:
- [ ] Please try as much as possible to stick to the Figma design. I have noticed that the `size` used on this paragraph is much different from the one of the Figma. Kindly modify this size to `15px` as it is in Figma design ☺️